modern/src/App.js

```diff
@@ -22,33 +22,81 @@ async function getSkin() {
   return await initialize(zip, skinXml);
 }
 
-function handleMouseEventDispatch(node, event, eventName) {
-  const rect = event.target.getBoundingClientRect();
-  const x = event.clientX - rect.left;
-  const y = event.clientY - rect.top;
+function setupUpdates(node) {
+  const [ignored, forceUpdate] = useReducer(x => x + 1, 0);
+  let once = true;
+  node.js_listen("js_update", () => {
```

Comment:
This probably needs to go inside a `useEffect`.

Reply:
oh yea, thats why I "needed" that `once` hackery :-X, thanks!